apps/frontend/src/pages/not-found/styles.module.css

```
@@ -0,0 +1,54 @@
+.container {
+	display: flex;
+	flex-grow: 1;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	height: 100%;
+	font-family: var(--font-family);
+}
+
+.error-container {
+	display: flex;
+	flex-direction: column;
+	gap: 64px;
+	align-items: center;
+	justify-content: space-between;
+	width: 360px;
+	height: 312px;
```

Avoid fixed height, there is no need for it, and it will make responsiveness implementation much harder. And the same for other styles in this file 

```suggestion
	width: 360px;
```